src/utils/FontManager.js

```diff
@@ -77,9 +77,10 @@ export async function fixupColorFonts() {
     }
 
     if (await isColrFontSupported()) {
-        const font = new FontFace("Twemoji",
-            `url('${require("../../res/fonts/Twemoji_Mozilla/TwemojiMozilla-colr.woff2")}')`, {});
-        document.fonts.add(font);
+        const path = `url('${require("../../res/fonts/Twemoji_Mozilla/TwemojiMozilla-colr.woff2")}')`;
+        document.fonts.add(new FontFace("Twemoji", path, {}));
```

Comment:
add a comment maybe why we are doing this?

Reply:
Added!